README.md

```diff
@@ -37,31 +37,35 @@ both for guidelines on code formatting and git usage.
 
 ### Modellers, texture and GUI artists, and Sound Engineers
 To work on the art assets you will want a working copy of the game.
-You can find official releases here: http://revolutionarygamesstudio.com/releases/
+You can find official releases [here][releasespage]
 alternatively you can request a newer version from developers, or compile the project yourself.
```

Comment:
This could be "official releases here, alternatively" instead of "official releases here alternatively".